database/abuseemail.go

```diff
@@ -180,10 +193,18 @@ func (a AbuseEmail) String() string {
 
 	// write response template
 	sb.WriteString("\nResponse Template:\n\n")
-	sb.WriteString(a.response())
+	sb.WriteString(a.Response())
 	return sb.String()
 }
 
+// Success is a small helper method that indicates whether the abuse email was
+// handled successfully, which means that links were found and all of the links
+// were blocked
+func (a AbuseEmail) Success() bool {
+	blocked, unblocked := a.result()
+	return len(blocked) > 0 && len(unblocked) == 0
```

Comment:
Is there a chance that an abuse email has 0 blocked and 0 unblocked because it simply didn't contain any links?

Reply:
Yeah, in which case I don't want to send an automated reply, I think the PR description mentions it. For the time being I want the definition of a successful abuse email to be that it detected abusive skylinks and was able to properly handle all of them.

Which I personally think makes sense because now we scan all incoming emails manually anyway, so for the time being we only want to reply in case we were able to take action and that action was successful.